src/stories/commits/CommitList.stories.mdx

```diff
@@ -0,0 +1,22 @@
+import { Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Commits/List/Intro" />
+
+# CommitList
+
+**Shows a history of changes on a task.**
+
+## Overview
+
+The **CommitList** component is a data table that shows the GitHub commit
+history of a **task**. This makes it possible to see at a glance how a task has
+changed over time.
+
+Each commit in the list features an id number, an image of the commit author,
+the commit message, and a timestamp of when the commit was made.
+
+<Story id="commits-list-component--component" />
+
+## Guidelines
+
+- _[To be added]_
```

Comment:
In this PR, or in a new PR?

Reply:
@jgerigmeyer I wasn't sure if there were other notable things to mention about this component at this stage. If there isn't, I can get rid of this section.